xarray/core/accessors.py

```diff
@@ -258,3 +258,24 @@ def round(self, freq):
             Array-like of datetime fields accessed for each element in values
         '''
         return self._tslib_round_accessor("round", freq)
+
+    def strftime(self, date_format):
+        values = self._obj.data
+        values_as_series = pd.Series(values.ravel())
+        strs = values_as_series.dt.strftime(date_format)
+        return strs.values.reshape(values.shape)
+    strftime.__doc__ = '''
```

Comment:
I may be missing something but what's the rational for assigning the doc string this way, as apposed to under the method definition?

Reply:
It came from pandas, and I think the purpose was to avoid the long line caused by the URL...maybe? I'm happy to adjust this to something more straightforward.

Reply:
I would lean toward just writing the docstring in the normal way, and silencing the lint error.